### ISSUE TITLE
Request - Update npm package to include latest changes

### ISSUE THREAD

**Issue by JonShort** [pushtell/react-bootstrap-date-picker#139]
<h1>Table of Contents</h1>

- [Expected Behavior](#expected-behavior)
- [Current Behavior](#current-behavior)
- [Possible Solution](#possible-solution)
- [Steps to Reproduce (for bugs)](#steps-to-reproduce-for-bugs)
- [Context](#context)
- [Your Environment](#your-environment)

## Expected Behavior
Version of the npm package should be updated from 5.1.0 (current) to 5.1.1

## Current Behavior
Currently, version 5.1.0 contains a few bugs, most notably the proptypes warnings that are thrown when the component is used in React 15.

## Possible Solution
Updating the npm package to 5.1.1 will include the following bugfixes:
- #131 - Fix to duplicate key warning when using minDate or maxDate
- #133 - Fix to remove proptypes warnings (now more important given React 16 is releasing soon)
- #137 - Fix to bug when using minDate or maxDate

As a further enhancement, the updating of the npm package could be added to the CI build, triggered when version patched. But this is not necessary to address this issue.

## Steps to Reproduce (for bugs)
N/A

## Context
This component is throwing proptypes warnings in unit tests & development in projects me & my team are working on.

I am also aware that members of the community have had to resort to using their own forks to fix the bugs associated with minDate and maxDate.

## Your Environment
N/A

**Comment by noushad-pp** [comment]
When will this npm package be upadated?

**Comment by vivek1623** [comment]
plz fix it soon, i am facing same issues.

**Comment by juanpaco** [comment]
React 16 has landed, and the PropTypes issue prevents upgrading to React 16.

**Comment by aanguswilliams** [comment]
@wehriam is there anything we can do to help get these fixes published?

**Comment by JonShort** [comment]
For those struggling with the PropTypes warnings, I've added a CI config to my fork of this repo to publish an npm package with the latest changes.

This is for my own personal use, and **i recommend not using it**, but if you really need to fix the propTypes warnings feel free to use [my npm package](https://www.npmjs.com/package/jonshort-shim-react-bootstrap-date-picker).

I'm hoping it'll be a temporary solution for me.

**Comment by flygis** [comment]
Please update the npm package so we all can move on with our life. The wait has been long enough for this.

**Comment by redheadedstep** [comment]
Well, I just wasted 4 hours tracking down #131 only to discover that the npm package hasn't been updated from 5.1.0 to 5.1.1.  Can you just run the command line, update npm, and save us all a lot of headache!?

Other than that, great tool and very easy to implement.

**Comment by travellingprog** [comment]
I know it's not an ideal solution, but we got past this by just pointing to the latest commit of this repo. In our package.json, we have react-bootstrap-date-picker set to `"github:pushtell/react-bootstrap-date-picker#f622dc8872dd180324ba5d3c295d009848f5814b"`. An NPM or Yarn install will be able to understand that.

**Comment by asazernik** [comment]
@pushtell Have you seen this?

**Comment by Crisfole** [comment]
Bump. At very least we need min/max date bugfix.

**Comment by filipetedim** [comment]
Bump. It's literally running one line. Maybe something happened to @pushtell ?

**Comment by winstonewert** [comment]
Do we need to fork?

**Comment by JonShort** [comment]
Yeah unfortunately you'll need to fork it, or use one of the forks that others have made 👍

**Comment by winstonewert** [comment]
Has anyone tried to put together a fork which aims to supplant the original? Or is everyone just making a bunch of random forks?
